uber/templates/accounts/homepage.html

```diff
@@ -36,7 +36,7 @@
     </ul>
 {% endif %}
 
```

Comment:
People with at-the-door reg access had access to these places but didn't see the links.